src/pages/Lending/Pool/components/Borrow/BorrowConfirm.tsx

```diff
@@ -244,24 +242,17 @@ export const BorrowConfirm = ({
 
   const isLendingQuoteSuccess = Boolean(confirmedQuote)
 
-  const memo = useMemo(() => {
-    if (!confirmedQuote) return
-    const supportedEvmChainIds = getSupportedEvmChainIds()
-    return supportedEvmChainIds.includes(fromAssetId(collateralAssetId).chainId as KnownChainIds)
-      ? toHex(confirmedQuote.quoteMemo)
-      : confirmedQuote.quoteMemo
-  }, [collateralAssetId, confirmedQuote])
-
```

Comment:
Nice one, indeed no hexlified memo unless we do EVM sends... which we shouldn't anymore!